storage/innobase/row/row0upd.cc

```diff
@@ -3168,6 +3137,7 @@ static bool row_upd_check_autoinc_counter(const upd_node_t *node, mtr_t *mtr) {
 
 #ifdef WITH_WSREP
   que_node_t *parent = que_node_get_parent(node);
+  trx_t *const trx = thr_get_trx(thr);
```

Comment:
trx is used only once below in WITH_WSREP block. Maybe it would be better to move it there?

Reply:
The usage below is like 
```c++
  3167 #ifdef WITH_WSREP
  3168   else if (trx && wsrep_on(trx->mysql_thd) && err == DB_SUCCESS &&
  3169     ┆   ┆  !row_upd_parent_has_cascade(parent))
```
So, it is better we capture it once and resue it instead of calling `thr_get_trx()` twice.

Reply:
I see, please ignore.